scene/3d/skeleton_3d.cpp

```diff
@@ -865,6 +865,10 @@ void Skeleton3D::set_bone_pose_rotation(int p_bone, const Quaternion &p_rotation
 	const int bone_size = bones.size();
 	ERR_FAIL_INDEX(p_bone, bone_size);
 
+	if (!p_rotation.is_normalized()) {
+		WARN_PRINT_ED("The quaternion bone " + get_bone_name(p_bone) + " pose rotation must be normalized.");
+	}
+
```

Comment:
```suggestion
```

I recommend removing it to focus on PR for adding buttons for now.

If we are going to add checks, we should add some checks to the fundamental of the Skeleton, not only the Rotation check, but also the Scale of 0 check, etc.